minStack.py

```diff
@@ -0,0 +1,48 @@
+#Time Complexity : pop:O(1), top:O(1) , push:O(1), getMin: O(1)
```

Comment:
Time complexity is not O(1) as per code implemented in the problem

Reply:
The TC of push() is O(n)? Since I am calling getmin() everytime I am calling push() function. Please correct if I am wrong.